yoga/model/assimp.cpp

```diff
@@ -47,6 +46,9 @@ void assimp_import_from_bytes(char* bytes_in, int length_in, int optimization_fl
     if (optimization_flags_in & OPTIMIZATION_FLAG_MESHES) {
         flags |= aiProcess_OptimizeMeshes;
     }
+    if (optimization_flags_in & FIX_FLAG_NORMALS) {
```

Comment:
The `optimization_flags_in` parameter may be renamed to `flags_in` as this is no more optimization flags only